client/blockchain-service/src/utils.rs

```diff
@@ -442,15 +456,15 @@ impl BlockchainService {
                     trace!(target: LOG_TARGET, "Forest root write task finished, lock is released!");
                     let state_store_context = self.persistent_state.open_rw_context_with_overlay();
                     state_store_context
-                        .access_value(&OngoingForestWriteLockTaskDataCf)
+                        .access_value(&OngoingProcessConfirmStoringRequestCf)
                         .delete();
                     state_store_context.commit();
                 }
                 Err(TryRecvError::Closed) => {
                     error!(target: LOG_TARGET, "Forest root write task channel closed unexpectedly. Lock is released anyway!");
                     let state_store_context = self.persistent_state.open_rw_context_with_overlay();
                     state_store_context
-                        .access_value(&OngoingForestWriteLockTaskDataCf)
+                        .access_value(&OngoingProcessConfirmStoringRequestCf)
                         .delete();
                     state_store_context.commit();
```

Comment:
In here, if we release the lock because a `submit_proof` task just finished, would we be deleting a potential `OngoingProcessConfirmStoringRequest` that is not actually happening?

Reply:
Not really since we are always writing the ongoing task when we create a new channel. This seems like it since no matter what task we delete the OngoingProcessConfirmStoringRequest, but in reality there can be only 1 ongoing task that writes the forest, so when the lock is for a SubmitProof this is a noop.

Reply:
Might be worth adding it in a comment